src/django_idom/utils.py

```diff
@@ -13,8 +13,19 @@
 from django_idom.config import IDOM_REGISTERED_COMPONENTS
 
 
-COMPONENT_REGEX = re.compile(r"{% *component +((\"[^\"']*\")|('[^\"']*'))(.*?)%}")
 _logger = logging.getLogger(__name__)
+_component_tag = r"component"
+_component_path = r"((\"[^\"']*\")|('[^\"']*'))"
+_component_kwargs = r"((.*?|\s*?)*)"
```

Comment:
While we're at it. Can we add names to these groups with `(?P<...>)`? Will just help out readability when matching.